ProductTree/ViewModels/ProductTreeViewModel.cs

```diff
@@ -135,6 +166,16 @@ public ProductTreeViewModel(Option option, ISession session, IThingDialogNavigat
             this.UpdateProperties();
         }
 
+        /// <summary>
+        /// Gets the value whether <see cref="ProductTreeViewModel"/> is option dependant
+        /// the value is used by the <see cref="CDP4CommonView.Views.BrowserHeader"/> to display or not the option informations
+        /// </summary>
+        public bool IsOptionDependant
+        {
+            get => this.isOptionDependant;
+            private set => this.RaiseAndSetIfChanged(ref this.isOptionDependant, value);
```

Comment:
I dont think this should be private for binding, or the mode has to be explicitly OneWay?.

Reply:
Not sure what this property is about... the ProductTree is always for a specific Option, so this property is not needed here.